IGV_automation/auto_1 copy.py

```diff
@@ -0,0 +1,63 @@
+import socket
+
+from pysam import VariantFile
+
+vcf_in = VariantFile(b"/Users/smgreen02/Documents/broad/klebs/results/vcf/frags_final_variants.vcf")  # auto-detect input format
+vcf_out = VariantFile('-', 'w', header=vcf_in.header)
+
+HOST = 'localhost'
+PORT = 60151
+
+#socket = IGVsocket(host='localhost', port=60151)
+s = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+s.connect((HOST, PORT))
+
+s.sendall(b'new \n')
+
+from_server = s.recv(4096)
+
+print(from_server)
+
+s.sendall(b'genome /Users/smgreen02/Documents/broad/klebs/klebs_new/reference/Klebsiella_pneumoniae_subsp_pneumoniae_MGH_78578_plasmids.fasta\n')
```

Comment:
Maybe an idea to create a function that sends a command to IGV and reads the response? Reduces duplicate code